ocp_utilities/operators.py

```diff
@@ -166,9 +168,21 @@ def install_operator(
             brew_token=brew_token,
             operator_market_namespace=operator_market_namespace,
         )
+    elif source_image:
+        source_name = f"catalog-{name}"
+        catalog_source = CatalogSource(
+            name=source_name,
+            namespace=operator_market_namespace,
+            display_name=source_name,
+            image=source_image,
+            publisher=source_name,
+            source_type="grpc",
```

Comment:
save and re-use (also exists under `create_catalog_source_for_iib_install`)

Reply:
@rnetser added function `create_catalog_source_from_image`